tests/neuro_san/zzz_hocons/test_integration_test_hocons.py

```diff
@@ -0,0 +1,53 @@
+# Copyright (C) 2023-2025 Cognizant Digital Business, Evolutionary AI.
+# All Rights Reserved.
+# Issued under the Academic Public License.
+#
+# You can be released from the terms, and requirements of the Academic Public
+# License by purchasing a commercial license.
+# Purchase of a commercial license is mandatory for any use of the
+# neuro-san SDK Software in commercial settings.
+#
+# END COPYRIGHT
+
+from unittest import TestCase
+
+import pytest
+
+from parameterized import parameterized
+
+from tests.framework.unittest.dynamic_hocon_unit_tests import DynamicHoconUnitTests
+
+
+class TestIntegrationTestHocons(TestCase):
+    """
+    Data-driven dynamic test cases where each test case is specified by a single hocon file.
+    """
+
+    # A single instance of the DynamicHoconUnitTests helper class.
+    # We pass it our source file location and a relative path to the common
+    # root of the test hocon files listed in the @parameterized.expand()
+    # annotation below so the instance can find the hocon test cases listed.
+    DYNAMIC = DynamicHoconUnitTests(__file__, path_to_basis="../../fixtures")
+
+    @parameterized.expand(DynamicHoconUnitTests.from_hocon_list([
+        # These can be in any order.
+        # Ideally more basic functionality will come first.
+        # Barring that, try to stick to alphabetical order.
+        "esp_decision_assistant/fallbacks_and_commondef_values.hocon",
+
+        # List more hocon files as they become available here.
+    ]))
+    @pytest.mark.integration
```

Comment:
Mark the above hocon file test cases as integration test only.
Now we have an integration to run, so those shouldn't fail.